packages/cdk-tweet-queue/lib/tweet-queue.ts

```diff
@@ -66,6 +74,7 @@ export class TweetQueue extends sqs.Queue {
       environment: {
         CREDENTIALS_SECRET: props.secretArn,
         TWITTER_QUERY: props.query,
+        TWITTER_QUERY_PARAMS: props.queryParams && JSON.stringify(props.queryParams),
```

Comment:
Use “??” instead of “&&”

Reply:
That's not what it's doing. Using nullish coalescing here would mean that we assign `props.queryParams` directly if it is nullish, but we want to stringify it first. Basically what I want is:

```
let queryParams = undefined
if (props.queryParams != null) {
  queryParams = JSON.stringify(props.queryParams)
}
```

Nullish coalescing would be equivalent to:

```
const queryParams = props.queryParams != null ? JSON.stringify(props.queryParams) : undefined
```

Reply:
Yes, I am stupid.

Reply:
Aren't we all 😂